packages/react/src/validationSchema.tsx

```diff
@@ -121,7 +121,7 @@ const validatorForField = (field: FieldMetadata) => {
     validator = (validator.nullable() as any).default(null);
   }
 
-  for (const validation of field.configuration.validations) {
+  for (const validation of field.configuration.validations ?? []) {
```

Comment:
I don't think this should have `?? []` because the GraphQL schema said it must return the validations. Having this fallback feels like a lie.

If you do this because of the introduction of the `gameCityModel` snapshot and the fields don't need any validations, you can add `validations: []` under the `configuration` object.

Reply:
Updated such that there is no more `??[]` here and added empty validation arrays to all of the fields in game.city